docs/typescript/activities.md

```diff
@@ -307,7 +316,27 @@ The [`Context.current().cancellationSignal`](https://typescript.temporal.io/api/
 <!--SNIPSTART typescript-activity-cancellable-fetch-->
 <!--SNIPEND-->
 
-## Local Activities
+## Advanced Features
+
+These are Activity features that most users will not need, but are available for advanced users.
+Please get in touch with us if you find the need for them.
+
+### Async Activity Completion
+
+Normally, an Activity is started and ended in the same Worker, for example a short HTTP call.
+However, sometimes you may want to record an Activity completion in a different process than when you started it.
+
+> If you are modeling human actions, we recommend using Signals rather than Async Activity Completion.
+> This is because Activities only have one timeout for the entire process and you won't know if the activity is timing out on a human or the process that kicks it off.
```

Comment:
```suggestion
> This is because Activities only have one timeout and if your Activity is split into two steps, one for kicking off the process (e.g. storing information in the DB), and one for human based resolution, it's best to use the timeout to detect failure in the former so it can be retried by the system.
```